frontend-knn/src/view/layout/home/send-input.form.tsx

```diff
@@ -159,9 +159,9 @@ export const SendInputForm: FC<ISendInputForm> = ({
                                         register={register}
                                         type="number"
                                         flexDir="row"
-                                        max={pwMaxFallback}
+                                        max={lengthAndWidthMax}
```

Comment:
The petal width and length fallbacks (plMaxFallback/pwMaxFallback) have been used to comply with the business rule restriction in which the petal dimensions can't be greather than sepal dimensions. Has it changed?